city/schemas.py

```diff
@@ -0,0 +1,24 @@
+from typing import Optional
+
+from pydantic import BaseModel
+
+
+class CityBase(BaseModel):
+    name: str
+    additional_info: str
```

Comment:
Consider making `additional_info` optional in the `CityBase` class if it's not always required, as it is optional in the `CityUpdate` class.